pseudoanonymize/common.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from dotenv import load_dotenv
+from joblib import Memory
+from openai import OpenAI
+
+load_dotenv()
+openai_api_key = os.getenv("OPENAI_API_KEY")
+client = OpenAI(api_key=openai_api_key)
```

Comment:
This is an unusual place to find this code 🤔 what is it doing here?

Reply:
Removed, I forgot to remove it while I was refactoring something😆